### PR TITLE
Validate pathtexts and shields without lines

### PR DESCRIPTION
**Pull request by pastk**

### CODE REVIEW

**Review by vng — APPROVED** [review]
Does python have some kind of min algorithm instead of sorted(xxx)[0] ?

**Comment by pastk** [comment]
> Does python have some kind of min algorithm instead of sorted(xxx)[0] ?

Right, changed it to `min(zooms)` 😊

File: src/libkomwm.py [hunk 1]
```diff
@@ -310,19 +310,37 @@ def add_zrange(first, last, result, maxzoom):
...
-                              f' for {cl}{object_id}')
-
+                    print(f'WARNING: non-contiguous visibility range {zoom_range} for {cl} {dr_type_comment}{object_id}')
+
+                dr_type = dr_type_comment[0]
+                icon_dr_type_comment = ('icon', None)
+                if (dr_type == 'caption' and icon_dr_type_comment in dr_types_comments and
+                    object_id in dr_types_comments[icon_dr_type_comment]):
+                        icon_zooms = sorted(dr_types_comments[icon_dr_type_comment][object_id])
+                        if sorted(zooms)[0] < icon_zooms[0]:
```
Comment (vng) [inline src/libkomwm.py]: min(zooms)?

Comment (pastk) [reply src/libkomwm.py]: fixed, thanks for noticing!

**Review by vng — APPROVED** [review]

**Pull request merged**